### PR TITLE
Re-scaffold forklift-operator with operator SDK v1.2.0 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by fbladilo**
This PR re-creates forklift-operator using latest Operator SDK (v1.2.0) , the project has been left intact for perfect inter-operability with SDK tooling. Please keep in mind many file locations and structure has changed. For reference see the following documentation : 

https://v1-2-x.sdk.operatorframework.io/docs/building-operators/ansible/reference/scaffolding/

Testing still undergoing.

### CODE REVIEW

File: forklift-operator-catalog.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ spec:
   displayName: Konveyor Forklift Operator
   publisher: Red Hat
   sourceType: grpc
-  image: quay.io/konveyor/forklift-operator-index:latest
+  image: quay.io/fbladilo/forklift-operator-index:latest
```
Comment (fdupont-redhat) [inline forklift-operator-catalog.yaml:10]: I guess it's temporary...

Comment (fbladilo) [reply forklift-operator-catalog.yaml:10]: Yes , will be removed once final

**Review by fdupont-redhat — APPROVED** [review]
LGTM

**Comment by fbladilo** [comment]
Closed in favor of #84

**Pull request closed without merging**